assets/sass/components/ads/_googlesitekit-ads-setup-module.scss

```diff
@@ -33,6 +33,15 @@
 			margin: 0;
 		}
 
+		.googlesitekit-text-field-conversion-tracking-id-prefix.mdc-text-field__icon {
+			top: 17.5px;
+		}
+
+		.mdc-text-field--error:not(.mdc-text-field--disabled, .mdc-text-field--focused) .mdc-floating-label {
+			font-size: $fs-body-md;
```

Comment:
The font-size of this element is apparently already 14px. Is there a reason we're resetting here?

Reply:
Yes you're correct, I've removed this in the [new PR here](https://github.com/google/site-kit-wp/pull/8469).